manifests/k/Kotatogram/Kotatogram/1.4.8/Kotatogram.Kotatogram.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS1.5-1-22000-282
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: Kotatogram.Kotatogram
+PackageVersion: 1.4.8
+PackageLocale: en-US
+Publisher: Kotatogram
+PublisherUrl: https://kotatogram.github.io/
+PublisherSupportUrl: https://kotatogram.github.io/faq/
+# PrivacyUrl: 
+Author: RadRussianRus
+PackageName: Kotatogram
```

This needs to match the `A`dd or `R`emove `P`rograms registry strings.
```suggestion
PackageName: Kotatogram Desktop
```

---

Resolved, I am sorry I always forgot to check these registry strings.